app/views/stations/new.erb

```diff
@@ -0,0 +1,69 @@
+<% @title = "Create New Station" %>
+
+<div class="add-new-station">
+<div class="jumbotron">
+  <div class="container">
+  <form class="edit" action="/stations" method="POST">
+  <h1 class="page-header">Add a Station
+    <a id="new-city" class="btn btn-warning btn-small" href="../../cities/new.erb">Add a New City</a>
+    <input type="submit" id="edit-large" class="btn btn-warning btn-small" value="Create"></h1>
+
+<div class="form group-row">
+  <div class="form-id">
+  <label for="input" class="col-xs-2 col-form-label">Name:</label></div>
+  <div class="col-xs-10">
+    <input class="form-control" type="text" name="station[name]" value="Name" onclick='this.value = "";'/><br/>
+  </div>
+  </div>
+
+<div class="form group-row">
+  <div class="form-id">
+  <label for="input" class="col-xs-2 col-form-label">Dock Count:</label></div>
+  <div class="col-xs-10">
+    <input class="form-control" type="integer" name="station[dock_count]" value="Dock Count" onclick='this.value = "";'/><br/>
+  </div>
+  </div>
+
+<div class="form group-row">
+  <div class="form-id">
+  <label for="input" class="col-xs-2 col-form-label">Latitude:</label></div>
+  <div class="col-xs-10">
+  <input class="form-control" type="integer" step="any" name="station[lat]" value="Latitude" onclick='this.value = "";'><br/>
+  </div>
+
+
+<div class="form group-row">
+  <div class="form-id">
+  <label for="input" class="col-xs-2 col-form-label">Longitude:</label></div>
+  <div class="col-xs-10">
+    <input class="form-control" type="integer" step="any" name="station[long]" value="Longitude" onclick='this.value = "";'><br/>
+    </div>
+
+
+<div class="form group-row">
+  <div class="form-id">
+  <label for="input" class="col-xs-2 col-form-label">Established:</label></div>
+  <div class="col-xs-10">
+  <input type="text" class="form-control" name="station[installation_date]" value="Installation Date" /><br/>
+  </div>
+  </div>
+</div>
+</div>
+
+
+<div class="form-id">
+  <label for="input" class="col-xs-2 col-form-label">City:</label></div>
+  <div class="col-xs-10">
+  <div class="dropdown">
+    <select id="drop-down-font" class="form_dropdown" name="station[city_id]">
+      <ul>
+            <option selected="selected" value="">-Select City-</option>
+      <% @cities.sort_by(&:name).each do |city| %>
```

Comment:
let's create a method that sorts our cities for us rather than doing it in the view. the view shouldn't know how to sort cities or even that it needs to sort cities